src/module/system/text-editor.ts

```diff
@@ -749,8 +741,8 @@ class TextEditorPF2e extends TextEditor {
             return null;
         }
 
-        const item = args.rollData?.item instanceof ItemPF2e ? args.rollData?.item : null;
-        const actor = (args.rollData?.actor instanceof ActorPF2e ? args.rollData?.actor : null) ?? item?.actor ?? null;
+        const item = args.rollData?.item ?? null;
+        const actor = args.rollData?.actor ?? item?.actor ?? null;
```

Comment:
I remember this was for a reason, but I can not recall it. We can gamble if you're feeling lucky.

Reply:
Not feeling lucky, but would be interested in tripping over it if it is needed so we can restore and also document why.